certified-connectors/QPP NG Integration Hub/apiDefinition.swagger.json

```diff
@@ -30,20 +30,20 @@
           "properties": {
```

Comment:
Kindly fix all the typos and description length issues.  Also please check the space after each period, comma and upper case, lower case letter after period and comma.

Reply:
Done